freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -259,6 +269,120 @@ def handle(self, event):
         db_event.transition(EventState.SKIPPED, msg)
         return []
 
+    @staticmethod
+    def _split_semver(version):
```

Comment:
I guess this is unused / replaced by semver Python package, right?

Reply:
Yes, you are right. I forgot to remove it.